config/formats.js

```diff
@@ -479,1359 +479,16 @@ exports.Formats = [
 		},
 	},
 	{
-		name: "[Gen 7] Type Illusion",
-		mod: "gen7",
-		ruleset: ['[Gen 7] OU'],
-		banlist: ["Shedinja"],
+		name: "[Gen 7] Purebred",
 		desc: [
-			"&bullet; Coded by flufi. Completed by Mewth",
-			"Type Illusion has finally left it's beta stage! Credit to Mewth for completing unfinished work.",
-			"Nickname a Pokemon a single typing and the Pokemon will change to that typing in battle.",
-			"To keep a Pokemon's default typing, don't give it a nickname.",
-			"&bullet; <a href=\"http://exiledps.boards.net/board/20/type-illusions\">Type Illusion Thread</a>",
+			"&bullet; Coded by flufi.",
+			"If the Pok*eacute;mon in your first and second slot are opposite genders, they will",
```

Comment:
Not sure that's the correct code for Pokémon

Reply:
Ill just assume you accidently hit * when you meant to hit &

Reply:
^